[2025-03-31T21:37:28.556+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-31T21:37:28.569+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sensor_data_pipeline.create_kafka_hdfs_sink manual__2025-03-30T16:38:15.319337+00:00 [queued]>
[2025-03-31T21:37:28.573+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sensor_data_pipeline.create_kafka_hdfs_sink manual__2025-03-30T16:38:15.319337+00:00 [queued]>
[2025-03-31T21:37:28.573+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-31T21:37:28.585+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_kafka_hdfs_sink> on 2025-03-30 16:38:15.319337+00:00
[2025-03-31T21:37:28.587+0000] {standard_task_runner.py:64} INFO - Started process 198 to run task
[2025-03-31T21:37:28.591+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'sensor_data_pipeline', 'create_kafka_hdfs_sink', 'manual__2025-03-30T16:38:15.319337+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp8g4z42ka']
[2025-03-31T21:37:28.593+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask create_kafka_hdfs_sink
[2025-03-31T21:37:28.632+0000] {task_command.py:426} INFO - Running <TaskInstance: sensor_data_pipeline.create_kafka_hdfs_sink manual__2025-03-30T16:38:15.319337+00:00 [running]> on host airflow
[2025-03-31T21:37:28.689+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='sensor_data_pipeline' AIRFLOW_CTX_TASK_ID='create_kafka_hdfs_sink' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T16:38:15.319337+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T16:38:15.319337+00:00'
[2025-03-31T21:37:28.690+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-31T21:37:28.696+0000] {pipeline.py:63} INFO - Creating Kafka-HDFS sink connector for topic 'sensores'
[2025-03-31T21:37:28.697+0000] {pipeline.py:82} INFO - Sending connector configuration to Kafka Connect REST API
[2025-03-31T21:37:28.734+0000] {pipeline.py:94} INFO - ℹ️ Connector already exists, continuing with existing connector
[2025-03-31T21:37:28.734+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-31T21:37:28.734+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-31T21:37:28.737+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sensor_data_pipeline, task_id=create_kafka_hdfs_sink, run_id=manual__2025-03-30T16:38:15.319337+00:00, execution_date=20250330T163815, start_date=20250331T213728, end_date=20250331T213728
[2025-03-31T21:37:28.768+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-31T21:37:28.789+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-31T21:37:28.792+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
